docs/studios/unity/unity-sdk/in-editor-replay.md

```diff
@@ -4,10 +4,61 @@ sidebar_label: 'Bot Replay'
 
 # Replaying Bot Behaviors in the Unity Editor
 
-Bots running on Regression Games save all states observed and actions sent, which can be used in the editor
-for playback. This feature, combined with our validation framework, allows you to find and debug
-specific errors within your game.
+Bots running on Regression Games save all states observed, all actions sent, and all validations performed.  This data can be loaded in the editor for visualization and playback in the scene. This feature, combined with our validation framework, allows you to find and debug specific errors within your game.
```

Comment:
Suggest adding that this is per game tick, to give some indication of how the data is formatted. Something like 

"Bots running on Regression Games record the observed game state, queued actions, and validation results for each tick."

Reply:
resolved